src/Datasource/ModelAwareTrait.php

```diff
@@ -80,7 +80,7 @@ protected function _setModelClass($name)
      *
      * @param string|null $modelClass Name of model class to load. Defaults to $this->modelClass
      * @param string|null $modelType The type of repository to load. Defaults to the modelType() value.
-     * @return object The model instance created.
+     * @return \Cake\ORM\Table The model instance created.
```

Comment:
@dereuromark I looked for an interface, but didn't see one for Table. Which one should it be?